shell/platform/darwin/macos/framework/Source/FlutterView.mm

```diff
@@ -110,6 +110,15 @@ - (BOOL)acceptsFirstResponder {
   return YES;
 }
 
+- (void)cursorUpdate:(NSEvent*)event {
+  // When adding/removing views AppKit will schedule call to current hittest view
+  // cursorUpdate: at the end of frame to determine possible cursor change. If
+  // the view doesn't implement cursorUpdate: AppKit will set the default (arrow) cursor
+  // instead. This would replace the cursor set by FlutterMouseCursorPlugin.
+  // Empty cursorUpdate: implementation prevents this behavior.
+  // https://github.com/flutter/flutter/issues/111425
+}
```

Comment:
Is there any way to write a test for this?

Reply:
That's a tricky one. We could test if FlutterView responds to the selector. Other than that - well, it gets complicated:

IIRC the method is called when CATransaction is committed. I had little luck getting some calls like this invoked by AppKit in headless environment (when the window is not actually shown). On top of that, macOS embedder tests currently crash when the tests span multiple run loop turns, because of scheduled AppKit callbacks for views that don't exist anymore (I think FlutterTextInputPlugin is one of them). I think there's an issue created for this already but I can't find it right now.

Reply:
Can you try asking @Hixie for a test exemption on Discord?  (see bot comment above)